devtools/doc/rpc.py

```diff
@@ -501,6 +504,10 @@ def handle_data(self, data):
             self.variant_parser.handle_data(data)
 
     def write(self, file):
+        if self.name == 'PoolTransactionReject':
```

Comment:
Just curious: why `PoolTransactionReject` special?

Reply:
It's an enum with associated fields.